source/docs/software/pathplanning/system-identification/viewing-diagnostics.rst

```diff
@@ -59,47 +69,34 @@ One of the most-common errors is an inappropriate value for the motion threshold
-
-To solve this, decrease the setting and re-analyze the data.
-
-Too Long of a Dynamic Test
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-.. image:: images/dynamic-test-too-long.png
-   :alt: Voltage domain plot with threshold too high
+.. image:: images/high-threshold.png
+   :alt: Quasistatic time-domain plot with velocity threshold too high
 
-If the dynamic test is run too long, it tends to accumulate data that will only skew the kA fit. This is noticeable with a large concentration of points towards the middle of the Voltage Domain Acceleration plot. This issue can be assisted by decreasing the :guilabel:`Test Duration` slider.
+While not nearly as problematic as a too-low threshold, a velocity threshold that is too high will result in a large "gap" in the acceleration-versus-velocity plot.
 
-.. image:: images/test-duration-slider.png
-   :alt: Voltage domain plot with threshold too high
+To solve this, decrease the velocity threshold and re-analyze the data.
 
-Magnetic Encoders Velocity Noise
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Noisy Velocity Signals
+^^^^^^^^^^^^^^^^^^^^^^
 
-.. image:: images/magencoder-emi-time.png
-   :alt: Time domain plot with mag encoder EMI
+.. note:: There are two types of noise that affect mechanical systems - signal noise and system noise.  Signal noise corresponds to measurement error, while system noise corresponds to actual physical motion that is unaccounted-for by your model (e.g. vibration).  If SysId suggests that your system is noisy, you must figure out which of the two types of noise is at play - signal noise is often easier to eliminate than system noise.
 
-.. image:: images/magencoder-emi-voltage.png
-   :alt: Voltage domain plot with mag encoder EMI
+.. image:: images/velo-noise.png
 
-Magnetic encoders such as the `CTRE Mag Encoder <https://newsite.ctr-electronics.com/srx-mag-encoder/>`__ and the `AndyMark magnetic encoder <https://www.andymark.com/products/am-mag-encoder>`__ are extremely popular in FRC.  However, a particular noise pattern has been observed when these encoders are used on robot drives, whose particular cause is not yet known.  This noise pattern is uniquely distinguished by significant velocity noise proportional to motor velocity, and is particularly common on the kit-of-parts `toughbox mini <https://www.andymark.com/products/toughbox-mini-options>`__ gearboxes.
+Many FRC setups suffer from poorly-installed encoders - errors in shaft concentricity (for optical encoders) and magnet location (For magnetic encoders) can both contribute to noisy velocity signals, as can inappropriate filtering settings on the RIO.  Encoder noise will be immediately visible in your diagnostic plots, as can be seen above.  Encoder noise is especially common on the `toughbox mini <https://www.andymark.com/products/toughbox-mini-options>`__ gearboxes provided in the kit of parts.
```

Comment:
```suggestion
Many FRC setups suffer from poorly-installed encoders - errors in shaft concentricity (for optical encoders) and magnet location (For magnetic encoders) can both contribute to noisy velocity signals, as can inappropriate filtering settings  Encoder noise will be immediately visible in your diagnostic plots, as can be seen above.  Encoder noise is especially common on the `toughbox mini <https://www.andymark.com/products/toughbox-mini-options>`__ gearboxes provided in the kit of parts.
```

Reply:
Inappropriate filter settings on the motor controllers can do this too.

Reply:
Agreed, that's why I suggested removing the "on the RIO". Why did you mark this as resolved if you agree but didn't make a change?

Reply:
I had misread it as a suggested addition instead of removal.